solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc

```diff
@@ -311,7 +311,5 @@ Example has been provided in `sample_techproducts_configs` to override content-t
 * SOLR-15124: Removed three core level admin API endpoints because they are already registered at the node level
 where they really belong: /admin/threads, /admin/properties, /admin/logging
 
-* SOLR-12336: Remove Filter, SolrFilter and SolrConstantScoreQuery
```

Comment:
Are those all internal? It feels like a significant API change for... somebody. Plugin writers?

Reply:
Yes.  Plugin writers will find out one way or another :-). Maybe they will read CHANGES.txt.  The audience of this page is users our users.